build/mvc-scenarios.yml

```diff
@@ -74,6 +74,8 @@ parameters:
     arguments: --scenario ApiCrudListProducts $(crudApiJwtJobs) --property scenario=ApiCrudListProductsJwtAuth
   - displayName: plaintext (JWT)
     arguments: --scenario NoMvcAuth $(crudApiJwtJobs) --property scenario=PlaintextRawJwt
+  - displayName: plaintext (JWT Asymmetric Key)
+    arguments: --scenario NoMvcAsymmetricAuth $(crudApiJwtJobs) --property scenario=PlaintextRawJwt
```

Comment:
```suggestion
    arguments: --scenario NoMvcAsymmetricAuth $(crudApiJwtJobs) --property scenario=PlaintextAsymmetricKey
```

Reply:
Should it be "PlaintextJwtAsymmetricKey"?

Reply:
It's just the text that appears in the dashboard, if it's a better description then yes.